compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/bindingpatterns/BLangRestBindingPattern.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
2021?

Reply:
We decided to go with 2020 since this file is created in that year.